editor/test/integration/save_data_test.clj

```diff
@@ -1304,7 +1304,7 @@
   ;; being overridden from a template node in one of the root-level files in the
   ;; save data test project. If you add a field to the NodeDesc protobuf
   ;; message, you'll either need to add a template override for it (we suggest
-  ;; you add it to `checked02.gui`, which hosts the majority of the template
+  ;; you add it to `checked01.gui`, which hosts the majority of the template
```

Comment:
Previously we separated all template override checks from the layout override checks into distinct `.gui` resources, because they didn't play well together when combined. Now we want to check more extensively that both layout and template overrides are saved when they are used in combination. We now have `checked01.gui`, that references `checked02.gui` as a template. `checked01.gui` overrides all properties on all node types, both in its own nodes and on nodes imported from `checked02.gui`. In addition, a Landscape layout overrides all these properties across both scenes to different values.